test/unit/enrichment/process-invoice-lines.test.js

```diff
@@ -1,13 +1,14 @@
-const { FUND_CODE } = require('../../mocks/values/fund-code')
-const { NET_DESCRIPTION, GROSS_DESCRIPTION } = require('../../mocks/values/description')
-const { SOURCE_SYSTEM } = require('../../mocks/values/source-system')
 
 jest.mock('../../../app/enrichment/enrich-invoice-line')
 const mockEnrichInvoiceLine = require('../../../app/enrichment/enrich-invoice-line')
 
 jest.mock('../../../app/enrichment/validate-invoice-line')
 const mockValidateInvoiceLine = require('../../../app/enrichment/validate-invoice-line')
 
+const { NET_DESCRIPTION, GROSS_DESCRIPTION } = require('../../mocks/values/description')
+const { SOURCE_SYSTEM } = require('../../mocks/values/source-system')
+const scheme = require('../../mocks/scheme')
+
 const processInvoiceLines = require('../../../app/enrichment/process-invoice-lines')
 
 const invoiceLine = require('../../mocks/payment-requests/invoice-line')
```

Comment:
If this is another mock maybe move it, and it's associated bits, above processInvoiceLines